_source/_posts/2022-09-08-react-fastify-postgres.md

```diff
@@ -0,0 +1,897 @@
+---
+layout: blog_post
+title: ""
+author: gabi-dombrowski
+by: contractor
+communities: [javascript]
+description: ""
+tags: []
+tweets:
+- ""
+- ""
+- ""
+image:
+type: conversion
```

Comment:
```suggestion
github: https://github.com/oktadev/okta-react-fastify-progres-example
type: conversion
```

Repo doesn't exist yet, but that's where it will be